app/src/main/java/org/simlar/logging/Lg.java

```diff
@@ -72,6 +72,7 @@ private static String createTag()
 
 		final int n = FILENAME_SIZE_MAX - fileName.length();
 		if (n > 0) {
+			//noinspection StringConcatenationInFormatCall
```

Comment:
I am quite unsure what happens here. `FILENAME_SIZE_MAX` looks more like the minimum size, because if `fileName.length()` exceeds it, the whole `fileName` is added...